agent/agent.go

```diff
@@ -29,6 +29,7 @@ var SSH_TPM_AGENT_ADD = "tpm-add-key"
 type Agent struct {
 	mu       sync.Mutex
 	tpm      func() transport.TPMCloser
+	op       []byte
```

Comment:
This should be a callback similar to how `pin` is implemented. It would allow us to write tests for this feature.